### ISSUE TITLE
Perform an accessibility audit

### ISSUE THREAD

**Issue by thisisdano** [GSA/digitalgov.gov#121]
Do we know where we have accessibility problems? I know there are resources within 18F and elsewhere to perform an accessibility audit — and this would probably be a good way to understand our current state and how to measure our progress.

I think we all agree that accessibility concerns are both important to our readers. Additionally, they're important drivers for other design, code, and user experience improvement. Let's make it a priority and take the first step.

#### todo
- [x] Learn who performs accessibility audits in 18F
- [x] Request an audit
- [x] Get audit results

#### create these issues as follow-up
- Create a scorecard by which we can measure our progress / Publish audit results
- Collect a list of methods for self-audit
- Collect list of methods to do some accessibility audits programmatically (like via Circle CI)
- Follow up with accessibility guild about publishing on digitalgov

**Comment by jeremyzilar** [comment]
Dropping this here for later: https://tenon.io/testNow.php?url=http://www.digitalgov.gov

**Comment by thisisdano** [comment]
Requested an accessibility review from @nickbristow at 18F here: https://github.com/18F/Accessibility_Reviews/issues/23

@jeremyzilar How can we use this as a jumping-off point for more accessibility-related case studies/guides/how-tos?

**Comment by jeremyzilar** [comment]
I am learning a lot about the Federal CrowdSource Mobile Testing Program  https://github.com/GSA/digitalgov.gov/issues/216 and how they do govt-wide testing around mobile. They use Open Opportunities to get people to create reports on mobile-friendlyness and get back a lot of interesting results. 

Thinking that we might be able to post an open-opportunity around accessibility.

**Comment by thisisdano** [comment]
@jeremyzilar The TTS Accessibility Guild is interested in working with and publishing with digitalgov — so we should follow up with that!

**Comment by thisisdano** [comment]
This audit should be done by Tuesday 11/21, and we can go from there!

**Comment by thisisdano** [comment]
from https://github.com/18F/Accessibility_Reviews/issues/23

- - -

I have compiled my results for the index page here, https://docs.google.com/spreadsheets/d/1fEOqYUJqQVhg4VaHI7t21nSIon2jSA-EvmBQfIkJukg/edit#gid=854558714

Anyone at GSA should be able to access that sheet.

The main issues i'm seeing are:
Contrast: the red color is not compliant on all backgrounds
Unique Links: "Read more" links should include which article they are referring to
Image Alts: The image links should describe the image if its relevant and the article title
Layout: The headings need to be reworked so they match the documents visual outline. IE sub elements of a heading should be a higher heading. h2 > h3 > h4 etc

I would recommend addressing these issues across all pages, and then running a quick scan using a tool such as HTML_Codesniffer, or AXE to verify all issues have been found.

Please feel free to reach out if you have any questions. Once you've addressed these issues, i'd be happy to take another look.
